src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -48,6 +57,8 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
   {
     def vantage = project.extensions.getByType(VantageExtension)
 
+    project.configurations.removeAll{it.name in doNotResolveConfigurations}
```

Comment:
I don't think we actually want to be mutating the configurations as part of this task, do we?